sdk/communication/Azure.Communication.CallingServer/src/Models/MediaStreaming/MediaStreamingAudio.cs

```diff
@@ -2,34 +2,40 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Text.Json;
 using System.Text.Json.Serialization;
 
-namespace Azure.Communication.CallingServer.Models.Streaming
+namespace Azure.Communication.CallingServer
 {
     /// <summary>
     /// Streaming audio.
     /// </summary>
-    public class MediaStreamingAudio
+    public class MediaStreamingAudio : MediaStreamingPackageBase
     {
+        internal MediaStreamingAudio(BinaryData data, DateTime timestamp, string participantId, bool isSilence)
+        {
+            Data = data;
+            Timestamp = timestamp;
+            Participant = new CommunicationUserIdentifier(participantId);
+            IsSilent = isSilence;
+        }
+
         /// <summary>
-        /// Base64 Encoded audio buffer data .
+        /// The audio data.
         /// </summary>
-        [JsonPropertyName("data")]
-        public byte[] Data { get; set; }
+        public BinaryData Data { get; }
```

Comment:
It is byte[] Data, not the BinaryData

Reply:
We received the comment from AzureBoard to have it as BinaryData.

Reply:
I see, according to current implementation its byte[] coming from backend. Though, I have setup meeting with Jeffrey next week where we will collect more feedback from him. We can check this in with current backend implementation which is byte[] and have an update if we end up with BinaryData ( since then we will have lot of change on backend anyways :) )

I don't think we should block this PR until next week.